src/util/dataUtil.ts

```diff
@@ -25,6 +26,20 @@ export function riderData2PointSeries(riderData: ISensorData[], xPropName: keyof
     .value() as IPoint[]; // asserting on type because `_.isNumber` can't type out the strings and nulls
 }
 
+export function analyticData2PointSeries(analyticData: IAnalytic[], xPropName: keyof IAnalytic, yPropName: keyof IAnalytic): IPoint[] {
```

Comment:
This looks like a copy/paste of `riderData2PointSeries`, but doesn't logically do anything different.

If you've made this because you get compile errors when you pass in an `IAnalytic`, you can probably change function signature or massage the types to be what you want. I'd also be okay with you making the signature more general at this point, where it takes `any` and returns` IPoint[]`